pkg/internal/apply/applier_integration_test.go

```diff
@@ -0,0 +1,188 @@
+// +build integration
+
+package apply
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	//hrclientset "github.com/fluxcd/helm-operator/pkg/client/clientset/versioned"
+	kraanscheme "github.com/fidelity/kraan/pkg/api/v1alpha1"
+	"github.com/fidelity/kraan/pkg/internal/layers"
+	hrscheme "github.com/fluxcd/helm-operator/pkg/client/clientset/versioned/scheme"
```

Comment:
Use `github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1` instead

Reply:
@richardcase - This one is mine.  Why that pkg instead of the other?